### PR TITLE
[summerospp]add fluentbit http plugin

### PR DESCRIPTION
**Pull request by sjliu1**
### Which issue(s) this PR fixes:

Fixes #

### Does this PR introduced a user-facing change?

```release-note

```

### Additional documentation, usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by benjaminhuo** [comment]
@sjliu1 You need to sign you commit

**Review by wenchajun — APPROVED** [review]

**Comment by sjliu1** [comment]
> @sjliu1 You need to sign you commit

Fixed.

File: apis/fluentbit/v1alpha2/plugins/input/http.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+	Port *int32 `json:"port,omitempty"`
+	// Specify the key name to overwrite a tag. If set, the tag will be overwritten by a value of the key.
+	Tagkey string `json:"tagKey,omitempty"`
+	// Specify the maximum buffer size in KB to receive a JSON message,default 4M.
+	// +kubebuilder:validation:Pattern:="^\\d+(k|K|KB|kb|m|M|MB|mb|g|G|GB|gb)?$"
+	BufferMaxSize string `json:"bufferMaxSize,omitempty"`
+	// This sets the chunk size for incoming incoming JSON messages.
+	//These chunks are then stored/managed in the space available by buffer_max_size,default 512K.
+	// +kubebuilder:validation:Pattern:="^\\d+(k|K|KB|kb|m|M|MB|mb|g|G|GB|gb)?$"
+	BufferChunkSize string `json:"bufferchunkSize,omitempty"`
```
Comment (benjaminhuo) [inline apis/fluentbit/v1alpha2/plugins/input/http.go:29]:
```suggestion
	BufferChunkSize string `json:"bufferChunkSize,omitempty"`
```

File: apis/fluentbit/v1alpha2/clusterinput_types.go [hunk 2]
```diff
@@ -57,6 +57,8 @@ type InputSpec struct {
 	Forward *input.Forward `json:"forward,omitempty"`
 	// OpenTelemetry defines forward  input plugin configuration
 	OpenTelemetry *input.OpenTelemetry `json:"openTelemetry,omitempty"`
+	// HTTP defines forward  input plugin configuration
+	HTTP *input.OpenTelemetry `json:"http,omitempty"`
```
Comment (benjaminhuo) [inline apis/fluentbit/v1alpha2/clusterinput_types.go:61]:
```suggestion
	HTTP *input.HTTP `json:"http,omitempty"`
```

File: apis/fluentbit/v1alpha2/plugins/input/http.go (hunk 1 above)
Comment (sjliu1) [reply apis/fluentbit/v1alpha2/plugins/input/http.go]: Fixed，thanks!

File: apis/fluentbit/v1alpha2/clusterinput_types.go (hunk 2 above)
Comment (sjliu1) [reply apis/fluentbit/v1alpha2/clusterinput_types.go]: Fixed，thanks!

**Review by benjaminhuo — APPROVED** [review]

**Pull request merged**